MaskRCNN/model_frcnn.py

```diff
@@ -150,9 +146,6 @@ def sample_fg_bg(iou):
         # ious[i] = print_runtime_tensor("ious[i]", ious[i], prefix=prefix)
         num_fg, num_bg, fg_inds, bg_inds = sample_fg_bg(ious[i])
 
-        num_fg = tf.identity(num_fg, name=f'dump_num_fg-{i}')
```

Comment:
I thought these were used to name these tensors to refer to them later. Are they not needed?

Reply:
Dumping tensors was a one-off thing. We no longer dump tensors and these tf.identities are just unused leftovers